frontend/src/components/interface/drawer/NavigationDrawer.vue

```diff
@@ -49,18 +41,48 @@
         },
 
         data() {
+            const logoutButton = { title: "Wyloguj", icon: "mdi-logout", onClick: this.logout, show: this.loggedIn };
+            const darkModeButton = {
+                title: "Tryb ciemny",
+                icon: "mdi-theme-light-dark",
+                onClick: this.toggleDarkMode,
+                show: true,
+            };
             return {
                 shouldShowRailVersion: true,
+                darkModeButton,
+                logoutButton,
                 buttons: {
                     project: [
-                        { title: "Nowy projekt", icon: "mdi-file-document-plus", onClick: this.createNewProject },
-                        { title: "Zapisz projekt", icon: "mdi-content-save", onClick: this.openSaveProjectModal },
-                        { title: "Otwórz projekt", icon: "mdi-folder-open", onClick: this.openLoadProjectModal },
-                        { title: "Kod debugujący", icon: "mdi-code-braces", onClick: this.showExtendedCode },
+                        {
+                            title: "Nowy projekt",
+                            icon: "mdi-file-document-plus",
+                            onClick: this.createNewProject,
+                            show: true,
```

Comment:
Może zamiast `show` zrobić `hidden` i ustawiać to jak chemy ukryć jakiś przycisk (czyli obecnie tylko dla przycisku wyloguj.